test/test_saemref.py

```diff
@@ -84,3 +89,20 @@ def test_saemref_running(Process, Service, Socket, Command, supervisor_service_n
 def test_saemref_sync_source_cronjob(Command):
     jobs = Command.check_output("crontab -u saemref -l")
     assert 'cubicweb-ctl source-sync --loglevel error saemref' in jobs
+
+
+@wait_saemref_started
+def test_saemref_client(Process, Service, Socket, Command, supervisor_service_name):
+    with TemporaryDirectory() as tempdir:
+        output = Command.check_output(
+            'python -m saemref_client skos-download http://localhost:8080 '
+            '25651/v1 -o {}'.format(tempdir))
+        assert glob(path.join(tempdir, '25651-v1', '*.xml')), \
+            '{}\n\nFolder content: {}'.format(output, os.listdir(tempdir))
```

Comment:
This cannot work since we are executing commands on remote system (docker), so tempdir exists only on local system not on docker.